src/Splat.NLog/NLogLogger.cs

```diff
@@ -98,6 +113,666 @@ public void Write(Exception exception, string message, Type type, LogLevel logLe
+        }
+
+        /// <inheritdoc/>
+        public void Fatal<TArgument1, TArgument2, TArgument3, TArgument4, TArgument5, TArgument6, TArgument7, TArgument8>(string messageFormat, TArgument1 argument1, TArgument2 argument2, TArgument3 argument3, TArgument4 argument4, TArgument5 argument5, TArgument6 argument6, TArgument7 argument7, TArgument8 argument8)
+        {
+            _inner.Fatal(CultureInfo.InvariantCulture, messageFormat, argument1, argument2, argument3, argument4, argument5, argument6, argument7, argument8);
+        }
+
+        /// <inheritdoc/>
+        public void Fatal<TArgument1, TArgument2, TArgument3, TArgument4, TArgument5, TArgument6, TArgument7, TArgument8, TArgument9>(string messageFormat, TArgument1 argument1, TArgument2 argument2, TArgument3 argument3, TArgument4 argument4, TArgument5 argument5, TArgument6 argument6, TArgument7 argument7, TArgument8 argument8, TArgument9 argument9)
+        {
+            _inner.Fatal(CultureInfo.InvariantCulture, messageFormat, argument1, argument2, argument3, argument4, argument5, argument6, argument7, argument8, argument9);
+        }
+
+        /// <inheritdoc/>
+        public void Fatal<TArgument1, TArgument2, TArgument3, TArgument4, TArgument5, TArgument6, TArgument7, TArgument8, TArgument9, TArgument10>(string messageFormat, TArgument1 argument1, TArgument2 argument2, TArgument3 argument3, TArgument4 argument4, TArgument5 argument5, TArgument6 argument6, TArgument7 argument7, TArgument8 argument8, TArgument9 argument9, TArgument10 argument10)
+        {
+            _inner.Fatal(CultureInfo.InvariantCulture, messageFormat, argument1, argument2, argument3, argument4, argument5, argument6, argument7, argument8, argument9, argument10);
+        }
+
+        /// <inheritdoc/>
+        public void Debug<T>(T value)
+        {
+            _inner.Debug(value);
+        }
+
+        /// <inheritdoc/>
+        public void Debug<T>(IFormatProvider formatProvider, T value)
+        {
+            _inner.Debug(formatProvider, value);
+        }
+
+        /// <inheritdoc/>
+        public void DebugException(string message, Exception exception)
+        {
+            _inner.Debug(exception, message);
+        }
+
+        /// <inheritdoc/>
+        public void Debug(Exception exception, string message)
+        {
+            _inner.Debug(exception, message);
+        }
+
+        /// <inheritdoc/>
+        public void Debug(IFormatProvider formatProvider, string message, params object[] args)
+        {
+            _inner.Debug(formatProvider, message, args);
+        }
+
+        /// <inheritdoc/>
+        public void Debug(string message)
+        {
+            _inner.Debug(message);
+        }
+
+        /// <inheritdoc/>
+        public void Debug<T>(string message)
+        {
+            _inner.Factory.GetLogger(typeof(T).ToString()).Debug(message);
```

Comment:
Would it be worthwhile using an MRU cache? for _inner.Factory.GetLogger(…)
i'll just mark the one of them. so don't need to click resolve 15 times :)

Reply:
Probably not until https://github.com/reactiveui/splat/issues/266 is fixed which is next on my to-do list. Along with fixing the tests for the full loggers which I will do tonight

Reply:
I will address #266 and come back to this.

Reply:
I've just raised a PR for 266